examples/step-58/doc/results.dox

```diff
@@ -0,0 +1,207 @@
+  result in an average that has nothing to do with the actual phase
+  angle, the `ComplexPhase` class just uses the *maximal* phase
+  angle encountered on each cell.
+
+With these modifications, the phase plot now looks as follows:
+
+<p align="center">
+  <img src="https://www.dealii.org/images/steps/developer/step-58.phase-cyclic.png"
+         alt="Phase of the solution at t=0.242, with a cyclic color map"
+         width="400">
+</p>
+
+Finally, we can generate a movie out of this. (To be precise, the video
+uses two more global refinement cycles and a time step half the size
+of what is used in the program above.) The author of these lines
+made the movie with VisIt,
+because that's what he's more familiar with, and using a hacked color map
+that is also cyclic -- though this color map lacks all of the skill employed by
+the people who wrote the posts mentioned in the links above. It
+does, however, show the character of the solution as a wave equation
+if you look at the shaded part of the domain outside the circle of
+radius 0.7 in which the potential is zero -- you can see how everytime
+one of the bumps (showing the amplitude $|\psi_h(\mathbf x,t)|^2$)
+bumps into the area where the potential is large: a wave travels
+outbound from there. Take a look at the video:
+
+@htmlonly
+<p align="center">
+  <iframe width="560" height="315" src="https://www.youtube.com/embed/nraszP3GZHk"
+   frameborder="0"
+   allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+   allowfullscreen></iframe>
+ </p>
+@endhtmlonly
+
+So why did I end up shading the area where the potential $V(\mathbf x)$ is
+large? In that outside region, the solution is relatively small. It is also
+relatively smooth. As a consequence, to some approximate degree, the
+equation in that region simplifies to
+@f[
+  - i \frac{\partial \psi}{\partial t}
+  + V \psi
+  \approx 0,
+@f]
+or maybe easier to read:
+@f[
+  \frac{\partial \psi}{\partial t}
+  \approx - i V \psi.
+@f]
+To the degree to which this approximation is valid (which, among other things,
+eliminates the traveling waves you can see in the video), this equation has
+a solution
+@f[
+  \psi(\mathbf x, t) = \psi(\mathbf x, 0) e^{-i V t}.
+@f]
+Because $V$ is large, this means that the phase *rotates quite rapidly*.
+If you focus on the semi-transparent outer part of the domain, you can
+see that. If one colors this region in the same way as the inner part of
+the domain, this rapidly flashing outer part may be psychedelic, but is also
+distracting of what's happening on the inside; it's also quite hard to
+actually see the radiating waves that are easy to see at the beginning
+of the video.
+
+
+<a name="extensions"></a>
+<h3>Possibilities for extensions</h3>
+
+<h4> Better linear solvers </h4>
+
+The solver chosen here is just too simple. It is also not efficient.
+What we do is just punt the matrix to a sparse direct solver in every
```

Comment:
> punt the matrix

Hmm... I don't know if some of our second/third/... language English readers are going to understand this expression.

Reply:
Good point -- it's an American Football reference where a "punt" is when you voluntarily give up the ball because you don't know anything better to do with it.